source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -0,0 +1,155 @@
+=========================
+Hooks Extension Framework
+=========================
+
+What is the Hooks Extension Framework?
+=======================================
```

Comment:
```suggestion
*****************************************************
```

Reply:
Thank you, done: https://github.com/openedx/docs.openedx.org/pull/599/commits/a88af4d9f0b50091f95ef1ac2a07c7d1d7bd1872